Cargo.toml

```diff
@@ -46,6 +46,7 @@ panic-halt = "0.2.0"
 [features]
 rt = ["stm32f0/rt"]
 stm32f042 = ["stm32f0/stm32f0x2"]
+stm32f072 = ["stm32f0/stm32f0x2"]
```

Comment:
I'd add stm32f072 compatability in a seperate PR. Even then, you're missing it in travis

Reply:
Should I remove this then?